texttospeech/list_voices/list_voices_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestListVoices(t *testing.T) {
+	testutil.SystemTest(t)
+
+	var buffer bytes.Buffer
+	err := ListVoices(&buffer)
+	if err != nil {
+		t.Error(err)
+	}
+	out := buffer.String()
```

Comment:
Nit: this is usually named `got` (in other tests, too).

Reply:
Renamed everywhere